crates/compiler/gen_dev/src/generic64/x86_64.rs

```diff
@@ -563,7 +563,7 @@ where
     let (base_offset, size) = storage_manager.stack_offset_and_size(&sym);
 
     if size - copied >= 8 {
-        for _ in (0..(size - copied)).step_by(8) {
+        for _ in 0..(size - copied) / 8 {
```

Comment:
turns out this is subtly incorrect for weird sizes. E.g. if `size = 3` then 4 bytes would be copied with the old implementation.